### PR TITLE
Remove integration test instructions

### PR DESCRIPTION
**Pull request by annashipman**
And point them to the website. These instructions were formerly duplicated across all projects. The only difference is the parameters required to test each tool.

cf: https://github.com/gds-operations/vcloud-tools/pull/208

This one's for bikeshedding if required. If it's fine, will raise ones across all other tools.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -137,14 +137,19 @@ Run the default suite of tests (e.g. lint, unit, features):
...
+The parameters required to run the vCloud Core integration tests are:
+
+````
+default: # This is the fog credential that refers to your testing environment, e.g. `test_credential`
+  vdc_1_name: # The name of a VDC
+  catalog: # A catalog
+  vapp_template: # A vApp Template within that catalog
+  network_1: # The name of the primary network
+  network_1_ip: # The IP address of the primary network
+````
```
Comment (ajlanghorn) [inline README.md]: Do we not need a storage profile here? I'm sure I was needing one when I was using them against Tibus.

**Comment by alexmuller** [comment]
@dcarley and I think this looks good :+1:

File: README.md (hunk 1 above)
Comment (alexmuller) [inline README.md]: Merged this on the basis that this is a separate issue because the parameters haven't been changed in this pull request.

**Comment by mattbostock** [comment]
:+1:

**Pull request merged**